runtime/runc_jailer.go

```diff
@@ -63,8 +63,14 @@ type runcJailerConfig struct {
 	CPUs           string
 	Mems           string
 	CgroupPath     string
+
+	// DriveExposePolicy defines how the jailer exposes files.
+	DriveExposePolicy string
 }
 
+const driveExposePolicyCopy = "copy"
+const driveExposePolicyBind = "bind"
```

Comment:
If we can use iotas here and match to enum in the proto we won't need to do the string validation. Would that make sense here?

Reply:
Yes. Removed on the last revision.